doc/devel/documenting_mpl.rst

```diff
@@ -410,6 +410,12 @@ Use simple single or double quotes when giving string values, e.g.
 
   If 'tight', try to figure out the tight bbox of the figure.
 
+  No ``'extra'`` literal quotes.
```

Comment:
Was this meant to be a title instead of part of the code block?

Reply:
No, this is intentional. Originally, I tried to put this in an inline lieteral:

~~~
The use of extra literal quotes around the text (````'extra'````) is discouraged.
~~~

However, it appears that four backticks followed by a single-quote or double-quote character are not interpreted correctly (in contrast followed by a letter, e.g. ` ````literal```` ` further above).

I would like to have the negative example shown somewhere, but did not manage to write it as an inline literal. So I just added it it to the block literal.